### ISSUE TITLE
Adding and Removing devices from DeviceGroup

### ISSUE THREAD

**Issue by anemitoff** [StephenStrickland/GcmSharp#4]
I see that you had code for doing this but it is commented out, not sure why.

PS. Thanks for sharing this project!

**Comment by StephenStrickland** [comment]
@anemitoff yes, school has been busy as of late. Now that midterms are over, I can get back to work on this project.
